todo-api/README.md

```diff
@@ -0,0 +1,88 @@
+# NodeJS Todo-API
+
+## Disclaimers
+
+This sample code is provided "as is" and is not a supported product of Ping. It's purpose is solely to demonstrate how the Ping JavaScript SDK can be implemented within a React application. Also, this is not a demonstration of React itself or instructional for _how_ to build a React app. There are many aspects to routing, state management, tooling and other aspects to building a React app that are outside of the scope of this project. For information about creating a React app, [visit React's official documentation](https://reactjs.org/docs/create-a-new-react-app.html).
```

Comment:
This app is vanilla JS, right?

Reply:
No, this is actually the Node.js app, but we should remove the React wording here as it's an API, so any client can use it, regardless of framework.

Reply:
This is being updated...forgot to remove the react bits and add the api. Shall be fixed soon...